specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-08-01-preview/containerInstance.json

```diff
@@ -596,7 +602,7 @@
                 },
                 "restartPolicy": {
                   "type": "string",
-                  "description": "- `always` Always restart\n",
```

Comment:
Currently always is the only option and defaulted, desc should update when r2c/never are implemented.